core/src/main/java/jenkins/model/ItemCategory/ItemCategory.java

```diff
@@ -0,0 +1,74 @@
+package jenkins.model.ItemCategory;
+
+import hudson.Extension;
+import hudson.ExtensionPoint;
+import hudson.model.ModelObject;
+import jenkins.model.Messages;
+
+/**
+ * A category for {@link hudson.model.Item}s.
+ */
+public abstract class ItemCategory implements ModelObject, ExtensionPoint {
```

Comment:
This was proposed by @rsandell. AFAIK,

```
A model object has a human readable name.
 
And it normally has URL, but this interface doesn't define one.
(Since there's so many classes that define the <tt>getUrl</tt> method we should have such one.)
```

The first point can be valid in this context. Not second.